### DISCUSSION TITLE
Finishing touches [Summary]

### DISCUSSION THREAD

**Discussion by DubskySteam** [DubskySteam/AQP#64] [Ideas]
### Clemens
* Endpunkte verbessern (Consistency, Controller Interface)
* Admin Panel verschönern (Swagger, Nightmode, MicroService tools, Disco Modus)
* WorldMap2D beenden [Done]
* RabbitMQ Service
-> Datenübertragung per Message Bus
-> Lose Kopplung
-> API Ausfall / Datensicherheit

**TL;DR:** Skalierbarkeit geht durch die Decke (Hoffentlich)

### Eric
* Die Probleme bei den Pages beheben
* Neue Mockups in Figma erstellen (LightMode/NightMode).
* Design nach MockUp umsetzen.
* Evtl. noch eine mobile und Webansicht hinzufügen.
* (?) Design aufhübschen

### Mathis
* Die Probleme bei den Pages beheben
* (?) Design aufhübschen

**Comment by Soprah** [comment]
### **Bis 19.12.23**
- Versuchen alle restlichen Probleme bei meinen Pages zu beheben (funktional/grafisch)
### **Ab 19.12.23**
- Neue Mockups in Figma erstellen (LightMode/NightMode).
- Design nach MockUp umsetzen.
- Vielleicht noch eine mobile und Webansicht hinzufügen.
